src/Utils/util.ts

```diff
@@ -93,4 +94,10 @@ export function earlierDateOrTimeToday(timestamp: string): string {
     const dialogTime = moment(timestamp)
     const isDialogCreatedToday = dialogTime.diff(endOfYesterday) >= 0
     return dialogTime.format(isDialogCreatedToday ? 'LTS' : 'L')
+}
+
+export function isNewActionUnique(new_action: CLM.ActionBase, actions: CLM.ActionBase[]): boolean {
+    const needle = stringify({ ...new_action, actionId: "", createdDateTime: "" })
+    const haystack = actions.map(action => { return stringify({ ...action, actionId: "", createdDateTime: "", suggestedEntity: action.suggestedEntity || null, packageCreationId: 0, packageDeletionId: 0, version: 0 }) })
```

Comment:
Put the properties on separate lines for better readability:

Example:
```
const haystack = actions.map(action => stringify({
    ...action,
    actionId: "",
    createdDateTime: "",
    suggestedEntity: action.suggestedEntity || null,
    packageCreationId: 0,
    packageDeletionId: 0,
    version: 0 }))
```

Reply:
Good suggestion